### DISCUSSION TITLE
Periodic Future calls

### DISCUSSION THREAD

**Discussion by kemalinnowise** [serverpod/serverpod#1063] [Q&A] [answered]
In my project on Serverpod I have to use FutureCalls twice a day. Like in 12 AM and 12 PM.
Can someone provide me with options ? I have no idea how can I achieve that.

**Comment by vlidholt** [comment, marked as answer]
Currently you can schedule a new call after your future call completes. Another option is to use a regular `Timer` to call a method at a specific time.

We should probably simplify this in the future.
